docfx/doxygen2yaml.cc

```diff
@@ -68,12 +68,14 @@ void CompoundRecurse(YAML::Emitter& yaml, YamlContext const& ctx,
   for (auto const child : node) {
     if (!IncludeInPublicDocuments(ctx.config, child)) continue;
     if (IgnoreForRecurse(child)) continue;
+    // Enums need to get their own files, never recurse in them.
+    // if (AppendIfEnum(yaml, ctx, child)) continue;
```

Comment:
Remnant?

Reply:
Kind of, I thought people would serve as "here is the code that you think may be needed". But it was a bad idea. Removed.